src/writeData/clients/Javascript/initialize.example

```diff
@@ -1,7 +1,7 @@
 const {InfluxDB} = require('@influxdata/influxdb-client')
 
 // You can generate an API token from the "API Tokens Tab" in the UI
-const token = '<%= token %>'
+const token = '<INFLUX_TOKEN>'
```

Comment:
to get the token from the system environment in node.js:
```suggestion
const token = process.env.INFLUX_TOKEN
```
FYI: The javascript client library also can also run in a browser or in Deno.js . The browser cannot get system environment vars, Deno.js would use `Deno.env.get('INFLUX_TOKEN')`